presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -128,17 +133,17 @@ default void rollbackTransaction(long transactionId)
         throw new UnsupportedOperationException();
     }
 
-    default boolean sendTransactionHeartbeatAndFindIfValid(long transaction)
+    default boolean sendTransactionHeartbeatAndFindIfValid(long transactionId)
```

Comment:
Please rename these parameters in the previous commit that adds the methods to the interface. (same for `ThriftMetastore`)